lib/beaker-pe/install/pe_utils.rb

```diff
@@ -1396,10 +1396,10 @@ def solaris_key_chain_fix
 EOM
           hosts.each do |host|
             if host.platform=~ /solaris-11(\.2)?-(i386|sparc)/
```

Comment:
Do we need x86_64 in here, or is `solaris11-64f` somehow i386?

Reply:
Totally ignorant here, Is i386 A reference vanagon? Like https://github.com/puppetlabs/vanagon/blob/80851591ef40c14b439d4a63a91f68d2d0526790/lib/vanagon/platform/defaults/solaris-11-i386.rb actually points to x86_64?

Ya I was on the fence about removing the (.2) since we hadn't actually destroyed the old pools yet.

Reply:
I don't remember why, but we use `solaris-11-i386` as the name of the `vanagon` platform to build on: https://github.com/puppetlabs/puppet-agent/blob/main/configs/platforms/solaris-11-i386.rb

And that is reflected in the beaker-hostgenerator data:

```
$ bx beaker-hostgenerator solaris11-64a 
---
HOSTS:
  solaris11-64-1:
    platform: solaris-11-i386
    template: solaris-11-x86_64
    hypervisor: vmpooler
    roles:
    - agent
```

I assume this step is trying to match `11` or `11.2` but not `11.4`. Presumably the cert doesn't need updating on that platform?

Reply:
This is an example of why I'm not a big fan of making the `solaris-11-x86_64` pool actually be `11.4`. There are probably lots of crusty assumptions built up that `solaris-11-x86_64` is < 11.4

Reply:
I guess I figured it was a good time to rip the bandaid off since 11.4 is the final release and 11.1 and 11.2 aren't supported anymore.

Reply:
Well, it looks like solaris11-64 has solaris-11-i386 underneath, but the vmpooler platform is solaris-11-x86_64.  Weird.

https://github.com/voxpupuli/beaker-hostgenerator/blob/69a61044edb40b763f8bcc3ca8048c84c41bee01/lib/beaker-hostgenerator/data.rb#L1198-L1205

Since we're interested in `platform`, looks like i386 is still fine.

Reply:
I would really like to get this if possible though so that nightly tests get fixed.

Reply:
Either way this change is backwards compatible, so I'll release it as 2.11.24